django/utils/timezone.py

```diff
@@ -299,13 +299,18 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
     """
     Converts an aware datetime.datetime to local time.
 
-    Local time is defined by the current time zone, unless another time zone
-    is specified.
+    Only aware datetimes are allowed. When value is omitted, it defaults to
+    now().
+
+    Local time is defined by the current time zone, unless another time zone is
```

Comment:
I wouldn't have reflowed this line since you didn't make any other changes and it would simplify the diff.